KomaMRICore/src/datatypes/Scanner.jl

```diff
@@ -1,14 +1,14 @@
 """
-    scanner = Scanner(B0, B1, Gmax, Smax, ADC_Δt, seq_Δt, GR_Δt, RF_Δt,
+    sys = Scanner(B0, B1, Gmax, Smax, ADC_Δt, seq_Δt, GR_Δt, RF_Δt,
         RF_ring_down_T, RF_dead_time_T, ADC_dead_time_T)
 
 The Scanner struct.
 
 # Arguments
-- `B0`: (`::Real`, `=1.5`, `[T]`) main magnetic field
+- `B0`: (`::Real`, `=1.5`, `[T]`) main magnetic field amplitude
 - `B1`: (`::Real`, `=10e-6`, `[T]`) maximum RF amplitude
-- `Gmax`: (`::Real`, `=60e-3`, `[T/m]`) maximum Gradient
-- `Smax`: (`::Real`, `=500`, `[mT/m/ms]`) maximum slew-rate
+- `Gmax`: (`::Real`, `=60e-3`, `[T/m]`) maximum gradient amplitude
+- `Smax`: (`::Real`, `=500`, `[T/m/s]`) maximum slew-rate for gradients
```

Comment:
maximum gradient's slew rate

Reply:
Also you changed the units of the slew rate from mT/ms/ms to T/m/s and is exactly the same, better to leave the first one.

Reply:
Edited!